### PR TITLE
Feature list

### PR DESCRIPTION
**Pull request by nonococoleo**
# Pull Request Template

in this PR, list api was added to service

## Checklist:

- [x] I have written test cases for this change, and my code passed the test
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

File: tests/test_service_routes.py [hunk 1]
```diff
@@ -166,6 +167,15 @@ def test_create_shopcart_with_pure_json(self):
         #    new_shopcart["update_time"], "Update time not set"
         #)
 
+    def test_get_pet_list(self):
+        """ Get a list of Pets """
```
Comment (KHafner) [inline tests/test_service_routes.py:171]: You missed changing from pet to shopcart here

Comment (nonococoleo) [reply tests/test_service_routes.py]: changed. that is embarrassing...

**Review by KHafner — APPROVED** [review]
looks good!

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**